src/rpc/rawtransaction.cpp

```diff
@@ -241,11 +303,7 @@ void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry)
             }
         }
         if (tx.nVersion >= ZIP225_TX_VERSION) {
-            UniValue orchard(UniValue::VOBJ);
-            CAmount valueBalanceOrchard = tx.GetOrchardBundle().GetValueBalance();
-            orchard.pushKV("valueBalance", ValueFromAmount(valueBalanceOrchard));
-            orchard.pushKV("valueBalanceZat", valueBalanceOrchard);
-            entry.pushKV("orchard", orchard);
+            TxOrchardBundleToJSON(tx, entry);
```

Comment:
And now that we will always have an `orchard` section, rework this to match the others:
```suggestion
            UniValue orchard = TxOrchardBundleToJSON(tx, entry);
            entry.pushKV("orchard", orchard);
```